lib/aem/aemdownload.js

```diff
@@ -99,12 +99,16 @@ class AEMDownload extends EventEmitter {
             }
         });
 
+        const retryOptions = {
+            retryMaxCount: 5
```

Comment:
Why 5?

Reply:
Based on the Nui logs we should at least retry twice, however in this case the code will be running on user desktops on customer networks. So I didn't want to define it too tightly.